code/modules/reagents/Chemistry-Reagents/Drug-Reagents.dm

```diff
@@ -56,56 +57,55 @@
 	color = "#60A584" // rgb: 96, 165, 132
 	overdose_threshold = 20
 	addiction_threshold = 10
+	metabolization_rate = 0.75* REAGENTS_METABOLISM
+	stun_threshold = 4
 
 /datum/reagent/drug/crank/on_mob_life(mob/living/M)
```

Comment:
It looks as though this and crank have very very similar on_mob_life definitions. Is there anything we can do to make it less repetitive?